### PR TITLE
Feature/chrome plugin

### PR DESCRIPTION
**Pull request by pwitusik**

### CODE REVIEW

File: G1ANT.Addon.Browser/Api/BrowserManager.cs [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                case "chrome":
+                    {
+                        var driverService = ChromeService.CreateService();
+                        driverService.StartingExtensionHtmlUrl = GetStartingExtensionHtmlPath();
+                        return new ChromeClient(driverService);
+                    }
+                case "firefox":
+                case "ff":
+                    break;
+                case "edge":
```
Comment (ChrisPrusik) [inline G1ANT.Addon.Browser/Api/BrowserManager.cs:69]: Add brave browser

File: G1ANT.Chrome.Driver/ChromeClient.cs [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace G1ANT.Chrome.Driver
+{
+    public class ChromeClient : BrowserClient
```
Comment (ChrisPrusik) [inline G1ANT.Chrome.Driver/ChromeClient.cs:13]: ChromiumClient?

File: G1ANT.Chrome.Driver/ChromeClient.cs [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        {
+        }
+
+        private bool urlLoaded = false;
+        public override void OpenBrowserWithUrl(string url, string extraArguments = "")
+        {
+            urlLoaded = false;
+            BrowserService.EventsService.OnTabUpdated += TabUpdatedHandler;
+
+            Process.Start("chrome.exe", $"\"{url}\" {extraArguments}");
```
Comment (ChrisPrusik) [inline G1ANT.Chrome.Driver/ChromeClient.cs:27]: Brave, Edge, etc.

**Review by ChrisPrusik — APPROVED** [review]

File: G1ANT.Addon.Browser/Api/BrowserManager.cs (hunk 1 above)
Comment (pwitusik) [reply G1ANT.Addon.Browser/Api/BrowserManager.cs:69]: can be added later

File: G1ANT.Chrome.Driver/ChromeClient.cs (hunk 2 above)
Comment (pwitusik) [reply G1ANT.Chrome.Driver/ChromeClient.cs:13]: Will be added during Edge and Brave implementation

File: G1ANT.Chrome.Driver/ChromeClient.cs (hunk 3 above)
Comment (pwitusik) [reply G1ANT.Chrome.Driver/ChromeClient.cs:27]: Will be added during Edge/Brave implementation

**Pull request merged**